### PR TITLE
powershell: patchelf replace liblttng-ust.so.0 to so.1 

### PR DESCRIPTION
**Pull request by p3psi-boo**
###### Motivation for this change

https://github.com/NixOS/nixpkgs/issues/138008

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/shells/powershell/default.nix [hunk 1]
```diff
@@ -47,6 +47,8 @@ stdenv.mkDerivation rec {
 
     ls $pslibs
   '' + lib.optionalString (!stdenv.isDarwin && !stdenv.isAarch64) ''
+    mkdir -p $out/lib
+    ln -s ${lttng-ust}/lib/liblttng-ust.so.1 $out/lib/liblttng-ust.so.0
```
Comment (jonringer) [inline pkgs/shells/powershell/default.nix:51]: I feel like the suffix is there for abi reasons, not sure if I'm comfortable feeding the binary lies.

Comment (p3psi-boo) [reply pkgs/shells/powershell/default.nix:51]: I think this PR can be merged for temporary needs. We can create a new PR after the Powershell upstream fixed the problem.

**Comment by Artturin** [comment]
once ~7.20 is removed the workaround can probably be removed https://github.com/PowerShell/PowerShell/pull/14688

**Comment by Artturin** [comment]
Sorry for hijacking your pr 😅

**Comment by SuperSandro2000** [comment]
@Artturin Can we upgrade directly?

**Comment by Artturin** [comment]
There's are preview release available. you'll have to ask the maintainers if they want to update to it 
https://github.com/PowerShell/PowerShell/releases/tag/v7.2.0-preview.9

**Comment by Artturin** [comment]
fixed aarch64 build

**Pull request merged**